package.json

```diff
@@ -28,65 +58,39 @@
     "lint-staged": "^13.2.3",
     "memfs": "^4.3.0",
     "prettier": "3.0.3",
+    "prettier-plugin-pkg": "0.18.0",
     "type-fest": "^4.7.1",
     "typescript": "~5.2.2"
   },
-  "optionalDependencies": {
-    "corepack": "0.20.0"
-  },
-  "engines": {
-    "node": "^16.20.0 || ^18.0.0 || ^20.0.0",
-    "npm": ">=7.0.0"
+  "overrides": {
+    "type-fest": "$type-fest",
+    "util": "^0.12.5"
   },
-  "scripts": {
-    "build": "npm run --workspaces --if-present build && npm run build:types",
-    "build:types": "tsc -b",
-    "clean:types": "tsc -b --clean",
-    "watch:types": "tsc -b --watch",
-    "lint:commit": "commitlint",
-    "lint:deps": "npm run --workspaces --if-present lint:deps",
-    "lint:eslint": "eslint .",
-    "lint:fix": "eslint . --fix",
-    "lint:staged": "lint-staged",
-    "lint": "npm run lint:eslint && npm run lint:deps",
-    "release:publish": "node scripts/publish.js --newPkg=${npm_config_newpkg}",
-    "release:dry-run": "node scripts/publish.js --dryRun --newPkg=${npm_config_newpkg}",
-    "release:rebuild": "npm run rebuild && npm run test:prep && npm run test:workspaces",
-    "release": "npm run release:rebuild && npm run release:publish --newPkg=${npm_config_newpkg}",
-    "rebuild": "npm run --workspaces --if-present rebuild && npm run rebuild:types",
-    "rebuild:types": "npm run clean:types && npm run build:types",
-    "setup": "husky install; npm run rebuild",
-    "test:prep": "npm run --workspaces --if-present test:prep",
-    "test:workspaces": "npm run --workspaces --if-present test",
-    "test:scripts": "ava --config scripts/ava.config.mjs",
-    "test": "npm run build && npm run test:prep && npm run test:workspaces && npm run test:scripts"
+  "ava": {
+    "files": [
+      "packages/*/test/*.spec.js",
+      "scripts/*.spec.js"
+    ],
+    "timeout": "2m"
   },
   "lint-staged": {
     "*.js": [
       "eslint --fix",
       "prettier --write"
     ],
-    "*.(ts|md|ya?ml)": [
+    "*.(ts|md|ya?ml|json)": [
       "prettier --write"
     ],
-    "!((package|policy)*).json": [
+    "!((package-lock|policy)*).json": [
       "prettier --write"
     ]
```

Comment:
think `package-lock.json` is ignored anyway right from

https://github.com/LavaMoat/LavaMoat/blob/a6bb55fcd092269fb6cf134aa6c723c03c5ff75e/.prettierignore#L16

Reply:
i guess same case for our couple policy files

https://github.com/LavaMoat/LavaMoat/blob/a6bb55fcd092269fb6cf134aa6c723c03c5ff75e/.prettierignore#L10-L11

so could remove this bit of the config (84-86)

but perhaps worth keeping so lint-staged doesn't run Prettier for sure